src/main/java/net/atos/zac/app/informatieobjecten/model/RESTEnkelvoudigInformatieobject.java

```diff
@@ -24,53 +25,74 @@ public class RESTEnkelvoudigInformatieobject {
 
     public UUID uuid;
 
+    @FormParam("identificatie")
```

Comment:
Just checking: all these `@FormParam` annotations do not break any existing functionality? I guess not because we have tests for these right? :-)

Reply:
Wondered the same, but:
* all tests we have pass
* these should matter only if the object is used in form submit